### DISCUSSION TITLE
Is it possible to specify something that is not synchronous (such as an action name or event name) on the edge?

### DISCUSSION THREAD

**Discussion by nanana-sudo** [UPPAALModelChecker/UPPAAL-Meta#122] [Q&A] [answered]
I'm really curious.

**Comment by yrke** [comment]
I'm not sure what you are asking about? Could you try to clarify your question, it is also often helpful if you can provide some context to what you are trying to achieve/model.

**Comment by nanana-sudo** [comment]
Sorry for making the question ambiguous.

My question is, I want to explicitly write the names of the asynchronous
actions to the edge in the editor.

**Comment by mikucionisaau** [comment, marked as answer]
Hi,
Perhaps you could use sending synchronization over a broadcast channel which is not received by any other process?
It's a common trick to make some edge urgent, but you also have a choice without urgency.

Best regards,
Marius
